pkg/scanners/template/scanner_template.go

```diff
@@ -0,0 +1,115 @@
+package main
```

Comment:
do we want to create a new repo for this?

Reply:
@sozercan do you mean create the template folder outside of pkg/scanners in pkg/template?

Reply:
i meant a new repo like `github.com/Azure/eraser-scanner-template` or something

example: https://github.com/open-policy-agent/gatekeeper-external-data-provider